soc/espressif/esp32c6/Kconfig.rtc

```diff
@@ -1,50 +0,0 @@
-# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
-# SPDX-License-Identifier: Apache-2.0
-
-if SOC_SERIES_ESP32C6
-
-choice RTC_CLK_SRC
```

Comment:
what is going on in this pr!? You add this file in one commit then remove it in another. Fix this pr